### PR TITLE
[DVCSMP-2959] MSA-2127: Fibaro Door/ Window Sensor v2 **Official Request by Fibaro"

### PR DESCRIPTION
**Pull request by SmartThings**
Submitting 'Fibaro Door/ Window Sensor v2 **Official Request by Fibaro"' for publication

### CODE REVIEW

**Comment by jackchi** [comment]
@parijatdas  Do you guys have this device in SRIB?

**Comment by parijatdas** [comment]
@jackchi We don't have this device. We only have the old Fibaro Door/Window Sensor.

**Comment by dkirker** [comment]
We have two in Mountain View.

**Comment by dkirker** [comment]
@jackchi Before this gets merged (or just following it) we will want to tweak a few things so they are consistent with the other 4 Fibaro ZW5 device types.

**Comment by dkirker** [comment]
I am looking this over today, I have a few changes that I will be adding.

File: devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+	definition (name: "Fibaro Door/Window Sensor 2", namespace: "fibargroup", author: "Fibar Group S.A.") {	
+		capability "Contact Sensor"
+		capability "Tamper Alert"
+		capability "Temperature Measurement"
+		capability "Configuration"
+		capability "Battery"
+		capability "Sensor"
+		capability "Health Check"
+	
+		attribute "temperatureAlarm", "string"
```
Comment (mckeed) [inline devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy:24]: Looks like `attribute "multiStatus"` should be here too

File: devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+						map.value = "clear"
+						map.descriptionText = "Tamper alert cleared"
+						break
+					case 2:
+						map.value = "overheat"
+						map.descriptionText = "Tamper alert: sensor removed or covering opened"
+						break
+					case 6:
+						map.value = "underheat"
+						map.descriptionText = "Tamper alert: sensor removed or covering opened"
```
Comment (mckeed) [inline devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy]: All the descriptionText values for temperature alarms are wrong

**Review by mckeed — CHANGES_REQUESTED** [review]

File: devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy (hunk 2 above)
Comment (dkirker) [reply devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy]: Ha! Good catch! Good ol' copy and paste...

**Comment by dkirker** [comment]
@mckeed Fixed! Thanks for catching the copy/paste mess!

**Review by mckeed — APPROVED** [review]
I haven't really reviewed the configuration sync system, but it can't really be worse than the way we usually do it.

**Comment by dkirker** [comment]
I jumped the gun. This isn't quite certified (I talked with the team last week and it should be there this week, I think).

**Pull request merged**